WTWSMS/map/geographical_region.txt

```diff
@@ -629,7 +629,7 @@ custom_volcanic = {
 	duchies = {
 		d_alexandria d_damietta d_cairo
 		d_mesopotamia d_osrhoene
-		d_mauria d_tingitana d_caesariensis d_ghana
```

Comment:
Why did you remove d_mauria?

Reply:
duplicate (it's inside the active_volcanic region already)

Reply:
Ah, I had not seen that.